pkg/util/kubelet/podresources/client.go

```diff
@@ -18,15 +18,107 @@ package podresources
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"sync"
 	"time"
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
+	"k8s.io/klog/v2"
 	v1 "k8s.io/kubelet/pkg/apis/podresources/v1"
 	"k8s.io/kubernetes/pkg/kubelet/util"
+
+	"github.com/kubewharf/katalyst-core/pkg/util/general"
+)
+
+const (
+	podResourcesClientTimeout              = 10 * time.Second
+	podResourcesClientMaxMsgSize           = 1024 * 1024 * 16
+	checkPodResourcesSocketInterval        = 100 * time.Millisecond
+	checkPodResourcesSocketTimeoutDuration = 10 * time.Second
 )
 
+var errNotImplemented = errors.New("pod resources client not initialized")
```

Comment:
rename errNotImplemented to errNotInitialized

Reply:
done